pom.xml

```diff
@@ -198,6 +197,12 @@
                 <artifactId>json</artifactId>
                 <version>20090211</version>
             </dependency>
```

Comment:
I think that because this is no longer in the BOM than the enterprise.gateway will fail when it picks up the next release of kaazing/gateway.  Did you build enterprise.gateway using kaazing/gateway develop-SNAPSHOT. 

I see no reason why this couldn't be in the BOM with a version range (if boms work like that).  Alternatively, you could update enterprise.gateway with a similar change, but that is duplicating code.